test/rules/no-exclusive-tests.js

```diff
@@ -26,43 +26,53 @@ ruleTester.run('no-exclusive-tests', rules['no-exclusive-tests'], {
     invalid: [
         {
             code: 'describe.only()',
-            errors: [ { message: expectedErrorMessage, column: 10, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 10, line: 1 } ],
+            output: 'describe()'
         },
         {
             code: 'describe["only"]()',
-            errors: [ { message: expectedErrorMessage, column: 10, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 10, line: 1 } ],
+            output: 'describe()'
         },
         {
             code: 'it.only()',
-            errors: [ { message: expectedErrorMessage, column: 4, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 4, line: 1 } ],
+            output: 'it()'
         },
         {
             code: 'it["only"]()',
-            errors: [ { message: expectedErrorMessage, column: 4, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 4, line: 1 } ],
+            output: 'it()'
         },
         {
             code: 'suite.only()',
-            errors: [ { message: expectedErrorMessage, column: 7, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 7, line: 1 } ],
+            output: 'suite()'
         },
         {
             code: 'suite["only"]()',
-            errors: [ { message: expectedErrorMessage, column: 7, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 7, line: 1 } ],
+            output: 'suite()'
         },
         {
             code: 'test.only()',
-            errors: [ { message: expectedErrorMessage, column: 6, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 6, line: 1 } ],
```

Comment:
We are reporting the start position of a `MemberExpression` property. For `test["only"]` we have a computed `MemberExpression` with the string literal `"only"` as the property. For `test.only` we have a non-computed `MemberExpression` with the identifier `only` as the property.

You can play around with the [esprima online demo](http://esprima.org/demo/parse.html) (don’t forget to enable the `Line and column-based` option).